util/MySqlMigrations/MySqlMigrations.csproj

```diff
@@ -19,15 +19,4 @@
   <ItemGroup>
     <Compile Include="..\EfShared\MigrationBuilderExtensions.cs" />
   </ItemGroup>
-
-  <ItemGroup>
-    <None Remove="Scripts\2021-09-21_01_SplitManageCollectionsPermission.sql" />
-    <None Remove="Scripts\2021-10-21_00_SetMaxAutoscaleSeatCount.sql" />
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Scripts\2021-09-21_01_SplitManageCollectionsPermission.sql" />
-    <EmbeddedResource Include="Scripts\2021-10-21_00_SetMaxAutoscaleSeatCount.sql" />
-    <EmbeddedResource Include="HelperScripts\2022-03-01_00_Up_MigrateOrganizationApiKeys.sql" />
-    <EmbeddedResource Include="HelperScripts\2022-03-01_00_Down_MigrateOrganizationApiKeys.sql" />
```

Comment:
I mentioned two files that are required, and these are where they are referenced. We use these as manual steps in an EF migration so that there won't be data loss.

Reply:
Added back.  Thank you!